src/assets/data/edition/series/1/section/5/op25/source-description.json

```diff
@@ -2249,55 +2249,55 @@
                                         "measure": "1",
                                         "system": "Klav.",
                                         "position": "5–6/8",
-                                        "comment": "Auf Rasur. Ante correcturam: Aufteilung der Noten auf die Systeme vermutlich wie in Textfassung 1."
+                                        "comment": "Auf Rasur. Ante correcturam: Aufteilung der Noten auf die Systeme vermutlich wie in <a (click)=\"ref.selectSvgSheet({complexId: 'op25', sheetId: 'M_317_TF1'})\">Textfassung 1</a>."
                                     },
                                     {
                                         "measure": "2",
                                         "system": "Ges.",
                                         "position": "2.–3. Note",
-                                        "comment": "Bogen rasiert (siehe Textfassung 1)."
+                                        "comment": "Bogen rasiert (siehe <a (click)=\"ref.selectSvgSheet({complexId: 'op25', sheetId: 'M_317_TF1'})\">Textfassung 1</a>)."
```

Comment:
Sind die "siehe Textfassung 1"-Verweise hier notwendig? Was ist damit gemeint? Sieht man dort die Rasur?

Reply:
Ich denke schon: Wir sind hier in "Korrekturen 2", die sich auf die Textfassungen der Werkedition von op. 25 beziehen. Das ist für M 317 Textfassung 2, wo z. B. der Bogen nicht mehr vorhanden ist. In Textfassung 1 ist er noch vorhanden. Noch wichtiger freilich in der Korrektur davor, wo Textfassung 1 'zeigt', was korrigiert wurde, ohne es hier umständlich beschreiben zu müssen.